src/ctapipe/reco/sklearn.py

```diff
@@ -386,6 +421,53 @@ def _predict_score(self, key, table):
     def _get_positive_index(self, key):
         return np.nonzero(self._models[key].classes_ == self.positive_class)[0][0]
 
+    def write(self, path, meta={}, overwrite=False):
```

Comment:
```suggestion
    def write(self, path, meta=None, overwrite=False):
```

No mutable default arguments.